docs/guidance/map/amendments.md

```diff
@@ -4,51 +4,55 @@
-* **Amendments**. For example, when the value or duration of a contract is changed. The term amendment often has a specific legal meaning for a publisher. Certain changes to a tender, award or contract might only be allowed as part of an amendment.
+* **Updates to existing information**. For example, to correct errors in earlier releases, or to make minor adjustments to titles, descriptions or dates.
+* **Amendments**. For example, when the value or duration of a contract is changed. The term amendment often has a specific legal meaning for a publisher. Certain changes to a tender, award or contract might only be permitted as part of an amendment.
 
 The nature of a change can be made explicit using:
 
-* **The release tag** (`tag`). For example, for a release with a new contract, use 'contract'. For an update to the contract, use 'contractUpdate', and for an amendment to the contract, use 'contractAmendment'.
+* **The release tag** field (`tag`), which is used to identify the type of change. For example, 'contract' identifies information about a new contract, 'contractUpdate' identifies an update to existing information about a contract, and 'contractAmendment' identifies a formal amendment to a contract.
 
-* **The amendments** building block. This can contain an array of amendment explanations, and clearly identify the releases that contain before and after values.
+* **The amendments** fields (`tender.amendments` and `contract.amendments`), which are used to list amendments along with their rationales and references to the releases that contain before and after values.
 
 ## Worked examples
 
 ### Example 1: Tender updates and amendments
 
-This example goes through updates during the **tender** stage in a contracting process.
+This example illustrates how new information, updates and amendments are modelled in OCDS.
 
-#### Tender
+#### Tender release
 
-A publisher issues a tender for a "Data merge tool". A new release with the `tag` 'tender' is built, see the JSON example below.
+A buyer publishes an opportunity for the purchase of office supplies.
 
 ```{jsoninclude} ../../examples/amendments/tender.json
 :jsonpointer: /records/0/releases/0
 :expand: tender, tag
 :title: Tender
 ```
 
-#### Tender Update
+#### Tender update release
 
-Weeks later, the publisher expands the `description` of the tender to provide more details about the tool being procured. A new release with the `tag` 'tenderUpdate' is built. The publisher does not consider this to be a formal 'amendment' to the tender, so does not publish any amendment information. See the JSON release below.
+The buyer now indicates the opportunity's main procurement category. The new information is not a formal amendment so the publisher uses the 'tenderUpdate' tag and omits the `tender.amendments` field.
+
+The publisher chooses to repeat fields whose values are unchanged from the previous release. Such fields can be omitted when a publication provides access to historic releases.
```

Comment:
Maybe make this an admonition, since access to historic releases is a bit tangential.

````suggestion
```{note}
The publisher chooses to repeat fields whose values are unchanged from the previous release. Such fields can be omitted when a publication provides access to historic releases.
```
````

Reply:
Ah, I see you had commented around this change earlier:

> Whilst the proposed change is more correct in terms of describing the data, I think the intent of the second clause in the original sentence was to highlight that Microsoft and Siemens are separate legal entities so I suggest instead linking to the description of Organization.id which explains how that field relates to organisation identifiers.

I think Microsoft and Siemens are sufficiently well known, that readers know they are separate entities without drawing attention to the construction of their identifiers.

Reply:
I think your second comment relates to https://github.com/open-contracting/standard/pull/1661/files#r1540402674, in which case I agree!

Reply:
Indeed - I must have mis-scrolled before pasting!